### ISSUE TITLE
weight of the verification loss and identification loss

### ISSUE THREAD

**Issue by JoyLuo** [D-X-Y/caffe-reid#2]
Dear @layumi @D-X-Y 
in paper named《A Discriminatively Learned CNN Embedding for Person Re-identificatio》，“
weight of the verification loss  = 1 and identification loss = 0.5 ” ,
but in  the code of @D-X-Y , Taking an example of train.proto of caffenet, the setting is opposite of the paper.

layer {
#this layer is calculate identification loss
  name: "loss"
  type: "SoftmaxWithLoss"
  bottom: "score"
  bottom: "label"
  top: "loss"
  **loss_weight: 1**
  propagate_down: true
  propagate_down: false
}

layer {
#this layer is calculate verification loss 
  name: "loss_dif"
  type: "SoftmaxWithLoss"
  bottom: "score_dif"
  bottom: "label_dif"
  top: "loss_dif"
  **loss_weight: 0.5**
  propagate_down: true
  propagate_down: false
}

**Comment by D-X-Y** [comment]
Yes, weight of the verification loss = 1 and identification loss = 0.5..
This seems a bug in my codes @layumi ....
I obtained similar performance on Alexnet/Googlenet/Resnet-50, but failed on VGG-16.
It might be caused by this bug, and I will check this soon.

**Comment by JoyLuo** [comment]
thanks for your kindly answer. and very sorry to that i have no time recently

**Comment by JoyLuo** [comment]
@D-X-Y 
Dear D-X-Y i want to check the performance on VGG-16 with the correct loss weights, please tell me the cause and effect of the failed on VGG-16

**Comment by D-X-Y** [comment]
I try to use the right loss weight, but the performance is quite bad. With the right loss weight, the other models' performance also change (see the updated readme)
